src/android/FirebaseMessagingPlugin.java

```diff
@@ -220,7 +220,13 @@ private void findChannel(String channelId, CallbackContext callbackContext) thro
             callbackContext.success(new JSONObject()
                 .put("id", channel.getId())
                 .put("name", channel.getName())
-                .put("description", channel.getDescription()));
+                .put("description", channel.getDescription())
+                .put("importance", channel.getImportance())
+                .put("badge", channel.canShowBadge())
```

Comment:
```suggestion
                .put("showBadge", channel.canShowBadge())
```

Reply:
The name `badge` was derived from the part where you set the channel properties:

`src/android/FirebaseMessagingPlugin.java:177`
```java
channel.setShowBadge(options.optBoolean("badge", true));
```

Shouldn't these be named the same?